src/main/hook.ts

```diff
@@ -109,20 +109,34 @@ ipcMain.on('start', async (event) => {
 		// Register key events
 		iohook.on('keydown', (ev: IOHookEvent) => {
 			const shortcutKey = store.get('pushToTalkShortcut');
-			if (keyCodeMatches(shortcutKey as K, ev)) {
+			if (!isMouseButton(shortcutKey) && keyCodeMatches(shortcutKey as K, ev)) {
 				event.reply('pushToTalk', true);
 			}
 		});
 		iohook.on('keyup', (ev: IOHookEvent) => {
 			const shortcutKey = store.get('pushToTalkShortcut');
-			if (keyCodeMatches(shortcutKey as K, ev)) {
+			if (!isMouseButton(shortcutKey) && keyCodeMatches(shortcutKey as K, ev)) {
 				event.reply('pushToTalk', false);
 			}
-			if (keyCodeMatches(store.get('deafenShortcut') as K, ev)) {
+			if (!isMouseButton(shortcutKey) && keyCodeMatches(store.get('deafenShortcut') as K, ev)) {
```

Comment:
This is checking for the push to talk shortcut key (again) in the deafen shortcut condition.

Reply:
This is done because if the shortcutkey is currently "MouseButton4" or other mouse button and it is passed into keyCodeMatches, it will definitely throw an error because it is not a valid Keyboard Mapping. 

So this check ensures that we are only passing into keyCodeMatches only if the shortcutkey is not a mouse button (which means it should be a keyboard button). Since deafen shortcut uses the same keyCodeMatches as pushToTalk, the check is also done here.

Reply:
My point is that the `shortcutKey` variable is still the push to talk key, the variable isn't updated to the deafen key so that condition is still inaccurate.
```suggestion
			if (!isMouseButton(store.get('deafenShortcut')) && keyCodeMatches(store.get('deafenShortcut') as K, ev)) {
```

Reply:
Ah thank you! I misunderstood what you said. Nice catch :)